packages/fx-core/src/component/feature/api.ts

```diff
@@ -44,7 +43,6 @@ export class TeamsApi {
     const functionName: string =
       (inputs?.[QuestionKey.functionName] as string) ?? DefaultValues.functionName;
     const actions: Action[] = [
-      LoadProjectSettingsAction,
```

Comment:
remove load/write project setting here. Is the project setting handled by core?

Reply:
Yes